tests/features/test_audio.py

```diff
@@ -114,6 +114,17 @@ def test_audio_decode_example_mp3(shared_datadir):
     assert decoded_example["sampling_rate"] == 44100
 
 
+@require_libsndfile
```

Comment:
(nit) I would rename it `require_libsndfile_with_opus` to make it explicit that it's just just checking if `libsndfile` is available, but also checking that the installed version does support OPUS
```suggestion
@require_libsndfile_with_opus
```